invenio_pidstore/models.py

```diff
@@ -69,6 +69,14 @@ class PIDStatus(object):
     should not be reused for something else.
     """
 
+    attributes = [
```

Comment:
We should probably go for Python 3.4 enums ... I will create a ticket if you haven't done it meantime.